var/spack/repos/builtin/packages/nmap/package.py

```diff
@@ -43,6 +43,7 @@ class Nmap(AutotoolsPackage):
     def configure_args(self):
         args = []
 
+        args.append("--disable-rdma")
```

Comment:
Might be worth adding a link to the upstream issue as a comment here for future adventurers.